conf/solr/rmlog

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
We like to add .sh to scripts: `rmlog.sh`

Reply:
Done.